pubspec.yaml

```diff
@@ -4,10 +4,11 @@ description: A simple Stripe API wrapper, that is meant to be used on the server
 homepage: https://github.com/enyo/stripe-dart
 documentation: https://github.com/enyo/stripe-dart/blob/main/README.md
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ^3.0.0
 dependencies:
   crypto: ^3.0.0
-  json_annotation: ^4.4.0
+  http: ^0.13.1
```

Comment:
Do we need http? As we use dio extensively here.

Reply:
I did not add the http dependency - it was added by Sparks1998* on 3/27 and I merely merged it in. I don't believe there is an actual dependency, so will remove it from the pubspec.yaml